src/main/java/com/artipie/Layout.java

```diff
@@ -100,12 +100,9 @@ public boolean hasDashboard() {
     }
 
     /**
-     * Flat layout. Consists of two parts.
+     * Org layout.
```

Comment:
@andreoss the second sentence is correct therefore it should be returned

Reply:
@genryxy Done.

Reply:
@genryxy Done.